tests/features/bootstrap/FeatureContext.php

```diff
@@ -18,6 +18,8 @@ class FeatureContext extends RawDrupalContext implements SnippetAcceptingContext
 
   use EntityTrait;
   use UtilityTrait;
+  // Uncomment to enable screen-shots during development.
+  // use \Drupal\joinup\Traits\ScreenShotTrait;
```

Comment:
Hm... Seems to me an over-engineering. But, yes, we can automate this. Let's not add now to much off-topic features. I needed because after owner change, a lot of Behat tests were broken and it's a nightmare to debug blindly. Let's open a follow up for an automation, if you think.